.github/workflows/supply-chain.yml

```diff
@@ -0,0 +1,32 @@
+name: supply-chain
+
+on:
+  push:
+    branches:
+     - main
+  pull_request:
+    branches:
+     - main
+  workflow_dispatch:
+
+jobs:
+  vet:
+    name: Vet Dependencies
+    runs-on: ubuntu-latest
+    env:
+      CARGO_VET_VERSION: 0.3.0
+    steps:
+    - uses: actions/checkout@v3
+    - name: Install Rust toolchain
+      uses: dtolnay/rust-toolchain@stable
+    - name: Cache cargo vet
```

Comment:
Is installing `cargo vet` so expensive that it's worth caching the binary? We don't do this for installing other Rust toolchain components, do we?

Reply:
It's not expensive in an absolute sense, but compiling the tool still takes a couple of minutes on GitHub infra, and it's a much better user experience to be notified of cargo-vet failures within a few seconds of submitting the PR.

Reply:
Toolchain components from rustup only require downloading and installing binaries, but `cargo install cargo-vet` does a release build locally, so it's worth it. I largely copied the cache action configuration from the `cargo vet` guide. The only other `cargo install` we do in CI is over in prio-server, for coverage tooling. We could save two and a half minutes there with the same approach.